helm/daq-config-server/values.yaml

```
@@ -48,9 +48,23 @@ readinessProbe:
     path: /healthz
     port: http
 
-volumes: []
+# Send logs to the dodal graylog stream for now until a more suitable stream exists
+logging_config:
+  graylog:
+    enabled: true
+    url: http://graylog-log-target.diamond.ac.uk:12232/
```

```suggestion
    url: http://graylog-log-target.diamond.ac.uk:12231/
```
Must: This port is wrong in blueapi if you want the dodal one